components/ModalSort.js

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+
+export default function ModalSort({
+  onSortComplexityStartHigh,
+  onSortComplexityStartLow,
+  onSortDurationStartLong,
+  onSortDurationStartShort,
+  onToogleSortModal,
+}) {
+  return (
+    <StyledBackground>
+      <StyledContainer>
+        <p>Sort projects by ...</p>
+        <StyledSortButton type="button" onClick={onSortDurationStartShort}>
+          <input
+            type="radio"
+            id="duration-short"
+            name="sort"
+            value="duration"
+          />
+          <label htmlFor="duration-short">Duration: short to long</label>
+        </StyledSortButton>
```

Comment:
See previous button comment.

Reply:
I understand that the button on the radio button is unconventional. But I am not sure how I trigger the sorting if I delete the button. This is where my function for sorting is.

Reply:
Could we use an `onChange` on the input element?